src/System.IO.FileSystem/tests/Directory/GetDirectories_str_str.cs

```diff
@@ -304,7 +304,7 @@ public static void runTest()
 
             strArr = Directory.GetDirectories(Path.Combine(".", dirName), "*BB*");
             iCountTestcases++;
-            if (strArr.Length != 2)
+            if (strArr.Length != (Interop.IsLinux ? 1 : 2)) // Linux is case-sensitive
```

Comment:
Mixed feelings.  We already have an issue open about being able to determine whether a given file system is case sensitive; at the moment in our libraries we're just hardcoding it to be case-sensitive for Linux and case-insensitive for Windows and OSX, but behind an abstraction that should make it relatively straightforward to fix later.  I don't feel strongly about it here in the tests, but I think I'd like to stick with the IsLinux checks for the time being, if for no other reason than some of the tests disabled due to casing are disabled via ActiveIssue attributes for a specific PlatformID, and we have no mechanism for having an ActiveIssue based on whether a file system is case sensitive, so keeping the platform name here at least keeps the tests internally consistent.  In the future if/when it becomes an issue, it should be fairly straightforward to sweep through.